README.md

```diff
@@ -1,59 +1,95 @@
+## How to use
+The component comes in two flavours:
+* Using classes `.icon .icon-species` declared in [`fonts.css` from the EBI Visual Framework](https://www.ebi.ac.uk/style-lab/websites/)
+* A bare-bones component that you can use in any web application without depending on the EBI VF styles being loaded in
+your environment
 
+The first one has the advantage that if the global CSS is changed, the component will then pick up the new styles. The second variant doesn’t need any dependency and can be used anywhere and it will just work. Notice that the second
+version still uses classes, but they will be uniquely scoped to the component
 
+## Instructions
 To install:
-`npm install --save react-ebi-species`
+```
+npm install --save react-ebi-species
+```
 
 To use as a React component:
 ```
 import EbiSpeciesIcon from 'react-ebi-species'
 ...
-<EbiSpeciesIcon species="homo sapiens"/>
+<EbiSpeciesIcon species={`homo sapiens`}/>
+```
+
+Alternatively, to use the “no class” flavour:
+```
+import { EbiSpeciesIconBare as EbiSpeciesIcon } from 'react-ebi-species'
+...
+<EbiSpeciesIcon species={`homo sapiens`}/>
 ```
 
 To use as a renderer and attach the icons to DOM elements:
 ```
-import {render} from 'react-ebi-species'
+import { render } from 'react-ebi-species'
 ...
-render({species: 'homo sapiens'}, 'id-of-your-DOM-element')
+render({ species: 'homo sapiens' }, 'id-of-your-DOM-element')
 ```
 
-To use directly in your browser (see `html/index.hml`):
+Or:
 ```
-<script src="dist/vendorCommons.bundle.js"></script>
+import { renderBare as render } from 'react-ebi-species'
+...
+render({ species: 'homo sapiens' }, 'id-of-your-DOM-element')
+```
+
+To use directly in your browser (see the source of the demo pages):
+```
+<script src="dist/vendors.bundle.js"></script>
 <script src="dist/ebiSpeciesIcon.bundle.js"></script>
+<script src="dist/ebiSpeciesIconBare.bundle.js"></script>
 
 <script>
-    ebiSpeciesIcon.render({species: 'homo sapiens'}, 'id-of-your-DOM-element')
+    ebiSpeciesIcon.render({ species: 'homo sapiens' }, 'id-of-your-DOM-element')
+    ebiSpeciesIcon.renderBare({ species: 'homo sapiens' }, 'id-of-your-DOM-element')
 </script>
 
 ```
 
-Made in the EBI for the [Expression Atlas](http://www.ebi.ac.uk/gxa).
+## Props
+The `species` prop should be a [scientific name](http://www.ensembl.org/info/about/species.html), although the names
+given in the EBI-Species documentation and other variants are available. It is case insensitive. Have a look at the
+demo page for details.
 
+|      Prop     |  Type  |       Default value         |                     Use case                     |
+|---------------|--------|-----------------------------|--------------------------------------------------|
+| `species`     | string | 🐰 oryctolagus cuniculus 🐰  | The name of the species to be rendered           |
+| `groupColors` | object | ```{ warmBlooded: `indianred`, plants: `mediumseagreen`, other: `deepskyblue` }``` | Colours for group of animals |
```

Comment:
Everything works well and looks fine. Just a very little text description for `groupColors` is not only for animals I think.

Reply:
Oh, you’re right. I’ll change it.